contracts/registries/ThirdPartyRegistry.sol

```diff
@@ -94,10 +95,9 @@ contract ThirdPartyRegistry is OwnableInitializable, NativeMetaTransaction {
     bool public initialThirdPartyValue;
     bool public initialItemValue;
 
-    event ThirdPartyAdded(string _thirdPartyId, string _metadata, string _resolver, bool _isApproved, address[] _managers, address _caller);
-    event ThirdPartyUpdated(string _thirdPartyId, string _metadata, string _resolver, address[] _managers, bool[] _managerValues, address _caller);
+    event ThirdPartyAdded(string _thirdPartyId, string _metadata, string _resolver, bool _isApproved, address[] _managers, uint256 _maxItems, address _caller);
```

Comment:
In order to keep the same variable name in the events handler I would use `uint256 _itemSlots` or something similar. Same for `ThirdPartyUpdated`

Reply:
Changed